webapp/portlet/src/main/webapp/space-settings/components/SpaceSettingApplications.vue

```diff
@@ -60,13 +60,27 @@ export default {
 
       this.$spaceService.getSpaceApplications(this.spaceId)
         .then(applications => {
-          this.applications = applications;
+          this.applications = applications.map(app => ({
+            applicationName: app.applicationName,
+            contentId: app.contentId,
+            description: this.$t(`SpaceSettings.application.${/\s/.test(app.displayName) ? app.displayName.replace(/ /g,'.').toLowerCase() : app.displayName.toLowerCase()}.description`),
+            displayName: this.$t(`SpaceSettings.application.${/\s/.test(app.displayName) ? app.displayName.replace(/ /g,'.').toLowerCase() : app.displayName.toLowerCase()}.title`),
+            id: app.id(),
+            removable: app.removable,
+          }));
         });
     },
     openDetail() {
       this.$spaceService.getSpaceApplications(this.spaceId)
         .then(applications => {
-          this.applications = applications;
+          this.applications = applications.map(app => ({
+            applicationName: app.applicationName,
+            contentId: app.contentId,
+            description: this.$t(`SpaceSettings.application.${/\s/.test(app.displayName) ? app.displayName.replace(/ /g,'.').toLowerCase() : app.displayName.toLowerCase()}.description`),
+            displayName: this.$t(`SpaceSettings.application.${/\s/.test(app.displayName) ? app.displayName.replace(/ /g,'.').toLowerCase() : app.displayName.toLowerCase()}.title`),
```

Comment:
why not having used the space portlet name instead, it will be simple to understand what to put as label in I18N of other addons.
Here the developers will have to make a computing switch display name of the application (replace "a space" by a "." & lowercase everything)
Wouldn't it be more simple to use `SpaceSettings.application.PORTLET_NAME.title` & `SpaceSettings.application.PORTLET_NAME.description` ?

Reply:
Yes you are right i agree